### ISSUE TITLE
Fix false positives for Sass functions in media-query-list-comma-newline-after

### ISSUE THREAD

**Issue by AlexWayfer** [stylelint/stylelint#4052]
> Clearly describe the bug

There are false positives for `media-query-list-comma-newline-after` when there is usage of some SCSS/SASS function.

> Which rule, if any, is the bug related to?

`media-query-list-comma-newline-after`

> What CSS is needed to reproduce the bug?

```scss
@media screen and (
  min-width: map-get($breakpoints, $breakpoint_from)
) and (
  max-width: map-get($breakpoints, $breakpoint_to) - 1px
) {
  @content;
}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```yaml
extends: stylelint-config-standard
```

> Which version of stylelint are you using?

`9.10.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint assets/styles/`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to SCSS functions.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
 28:34  ✖  Expected newline after "," in a multi-line list   media-query-list-comma-newline-after
 30:34  ✖  Expected newline after "," in a multi-line list   media-query-list-comma-newline-after
```

**Comment by jeddy3** [comment]
Closing [as stylistic rules are frozen](https://github.com/stylelint/stylelint/issues/5674). The community is welcome to migrate the rule [to a plugin](https://stylelint.io/developer-guide/plugins) and fix this bug.
